pubspec.yaml

```
@@ -7,21 +7,21 @@ environment:
   sdk: '>=2.12.0 <3.0.0'
 
 dependencies:
- typed_data: '^1.3.0'
- collection: '^1.15.0'
- hex: '^0.2.0'
- executor: '^2.2.2'
- logging: '^1.0.0'
- event_bus: '^2.0.0'
- string_scanner: '^1.1.0'
- date_format: '^2.0.2'
- build: '^2.0.0'
- yaml: '^3.1.0'
- path: '^1.8.0'
+  typed_data: '^1.3.0'
+  collection: '^1.15.0'
+  hex: '^0.2.0'
+  executor: '^2.2.2'
+  logging: '^1.0.0'
+  event_bus: '^2.0.0'
+  string_scanner: '^1.1.0'
+  date_format: '^2.0.2'
+  build: '^2.0.0'
+  yaml: '^3.1.0'
+  path: '^1.8.0'
+  dtls:
+    path: /home/sorunome/repos/famedly/frontend/dtls
```

@Sorunome Thank you for providing this PR! I guess now that the `dtls` package is [published on pub.dev](https://pub.dev/packages/dtls) this could be replaced? :)

```suggestion
  dtls: ^0.1.0
```